packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.tsx

```diff
@@ -116,7 +117,7 @@ export class AtomicCommerceInterface
    *
    * Will default to the value set in the Headless engine context if not provided.
    */
-  @Prop({reflect: true, mutable: true}) public language?: string;
+  @Prop({reflect: true, mutable: true}) public language: string = '';
```

Comment:
@louis-bompart Had to make this prop not optional otherwise I can't set a value for it in storybook. Do you know what could be causing this and if there is an alternative?

Reply:
So, there's two ways: the one you did, which can make sense tbh, as a 'reflect' param undefined doesn't make much sense imo, as this doesn't translate well in the HTML. (String, number, bools are the only one that realy does)

The other one is to tweak the argTypes in your story to override the argument definition in Storybook.
See https://storybook.js.org/docs/api/arg-types

Reply:
Thank you 💯 !